File: integrations/aws-strands/error-code-corpus/typescript/message_exact_whitespace.ts
/** A sentence whose own spacing and line break are what reach the wire. */
import { EventType, type BaseEvent } from "@ag-ui/core";

export function emit(): BaseEvent {
  return {
    type: EventType.RUN_ERROR,
    message: "Two  spaces, a\nline break, and a trailing space. ",
    code: "MESSAGE_EXACT_WHITESPACE",
  };
}

File: integrations/aws-strands/error-code-corpus/python/message_exact_whitespace.py
"""A sentence whose own spacing and line break are what reach the wire."""

from ag_ui.core import EventType, RunErrorEvent


def emit() -> RunErrorEvent:
    return RunErrorEvent(
        type=EventType.RUN_ERROR,
        message="Two  spaces, a\nline break, and a trailing space. ",
        code="MESSAGE_EXACT_WHITESPACE",
    )
